jac-uk/admin
# #1666 Stages and status & Merit List

## What's included?
This PR includes the following work:

- Merit List
- Stages & Status
- QT Platform integration

---

### Stages & Status
Introduces our new set of Stages and Statuses for applications.
Main ticket: #1666 
This PR replaces [#1673](https://github.com/jac-uk/admin/pull/1673)

Create or use an existing exercise which has applications.
Begin processing applications and navigate to Stages.

To test:
- Check that applications can be assigned the statuses and can progress through the amended stages
- Check that page size can be amended
- Check that search works
- Check that filter works

---

### Merit List
First iteration of Merit List supporting the following functionality:
 - Panellists
   - Add, edit, delete a central store of Panellists
 - Exercise
   - Configure capabilities/skills applicable to the exercise
   - Configure tools/methods used in shortlisting
 - Sift and Selection 
   - Initial set up including
     - Create panels
     - Add panellists to Panel
     - Allocate applications to Panels
     - Export to google drive
     - Key stats relevant to setting up the sift/selection
   - Run the sift/selection including:
     - View score sheet
     - Flag applications for moderation
     - Key stats relevant to running the sift/selection
   - Finalise and complete the sift/selection including
     - View grades and scores
     - Select new status after the sift/selection
     - Key stats relevant to finalising the sift/selection
     - Update status of all applications
 - Merit list
   - View grades and scores for all QT, Sift and Selection tasks
   - Expandable columns and fixed headers to help navigate large amounts of data 


## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
[ TBC ]

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
